include/cudaq/Optimizer/Transforms/Passes.td

```diff
@@ -828,6 +866,17 @@ def StatePreparation : Pass<"state-prep", "mlir::ModuleOp"> {
   ];
 }
 
+def StateValidation : Pass<"state-validation", "mlir::ModuleOp"> {
```

Comment:
Not sure about this name either. It seems to suggest that there is some sort of comparison going on, possibly to validate (verify) that an algorithm is correct.

Reply:
Renamed to `RemoveRuntimeFunctions`, let me know if you have a better naming idea